.travis.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     install: yarn install --cwd=dashboard --frozen-lockfile
     script:
     - yarn --cwd=dashboard run lint
-    - yarn --cwd=dashboard run test --maxWorkers=4 --coverage
+    - CI=true yarn --cwd=dashboard run test --maxWorkers=4
```

Comment:
why have we removed the --coverage flag from here?

Reply:
if i am not mistaken the `--coverage` flag was a hack so that the tests execute in a single run and exit. The CI=true has the intended effect and hence this change

Reply:
/cc @prydonius

Reply:
@sameersbn it actually wasn't, I wanted the builds to generate a coverage report. AFAIK the test runner automatically discovers CI environments. Can we revert this change?

Reply:
fixed in https://github.com/kubeapps/kubeapps/pull/370